macro-jwplayer-xip/pom.xml

```diff
@@ -48,7 +48,7 @@
         <plugin>
           <groupId>org.xwiki.commons</groupId>
           <artifactId>xwiki-commons-tool-extension-plugin</artifactId>
-          <version>${commons.version}</version>
+          <version>15.2</version>
```

Comment:
A comment to explain why a more recent version than the parent is needed would be interresting.

Reply:
Thanks @tmortagne I added a short comment. I hope it gives enough context about the motivation for using a more recent version.